hack/charts/installer/rolebased/Chart.yaml

```diff
@@ -1,7 +1,8 @@
 apiVersion: v1
 name: sagemaker-k8s-operator
 version: 0.1.0
-description: A Helm chart for deploying the SageMaker Operator for Kubernetes by assining permission using IAM role.
+appVersion: 1.0.1
```

Comment:
This will need to be updated for every release, right?

Reply:
Yes so when we release a new set of helm charts we'll update this version number and the tag on the image so that users can cross-reference all the values.